pkg/plugin/sdk/v1alpha2/plugin/grpc.go

```diff
@@ -84,6 +92,46 @@ func (m *GRPCServer) PostBuildHook(
 		m.Impl.PostBuildHook(req.IsInteractiveMode, prompter)
 }
 
+// CustomCommands translates the gRPC call to the Plugin CustomCommands
+// implementation. It returns a list of commands that the plugin implements.
+func (m *GRPCServer) CustomCommands(
+	ctx context.Context,
+	req *proto.CustomCommandsReq,
+) (*proto.CustomCommandsRes, error) {
+	customCommands, err := m.Impl.CustomCommands()
```

Comment:
Why do we proceed here and not return if there is an `err`?

Reply:
Fixed